bindings/python/dlite-collection.i

```diff
@@ -86,6 +107,30 @@ struct _DLiteCollection {
     dlite_instance_incref(inst);
     return inst;
   }
+
+  //%newobject value;
+  //%feature("docstring",
+  //  "Return the value for a pair of two criteria.
+  //
+  //  Useful if one knows that there may only be one value.
+  //
+  //  Parameters:
+  //      s, p, o: Criteria to match. Two of these must not be None.
+  //      d: If not None, the required datatype of literal objects.
+  //      fallback: Value to return if no matches are found.
+  //      any: If non-zero, return first matching value.
+  //
+  //  Returns:
+  //      The value of the `s`, `p` or `o` that is None.
+  //  ") value;
+  //const char *value(struct _DLiteCollection *coll,
+  //                  const char *s=NULL, const char *p=NULL,
+  //                  const char *o=NULL, const char *d=NULL,
+  //                  const char *fallback=NULL, int any=0) {
+  //  const char *v = triplestore_value(coll->rstore, s, p, o, d, fallback, any);
+  //  return (v) ? strdup(v) : NULL;
+  //}
+
```

Comment:
I think we should not keep commented out function in the code. 

One questino, though, do I red it correctly in that value search is moved from the core C code to the bindings in Python?

Reply:
Removed the commented out lines. 

No, value() is still only implemented in C. What I commented out here was my first attempt to interface it directly as a method of the Collection class. I gave that up and added it instead as Python code in dlite-collection-python.i using the low-level swig-generated function in dlite._dlite._collection_value().